UACloudLibraryServer/GraphQL/GraphTypes/AddressSpaceType.cs

```diff
@@ -36,7 +36,6 @@ public class AddressSpaceType : ObjectGraphType<AddressSpace>
         public AddressSpaceType()
         {
             Field(e => e.Title, type: typeof(NonNullGraphType<StringGraphType>));
```

Comment:
Should the AddressSpace also have an "identifier" (or id) field?

Reply:
An identifier is only associated with a nodeset file to uniquely identify it. The address space query just wraps it and adds metadata to it.

Reply:
In the GraphQL world the AddresSpace is an entity and should have an id, per convention. But let's resolve outside of this PR.